pkg/server/home_workspaces.go

```diff
@@ -291,79 +291,39 @@ func (h *homeWorkspaceHandler) ServeHTTP(rw http.ResponseWriter, req *http.Reque
 			// fall through because either not ready or RBAC objects missing
 		}
 
-		// Test if the user has the right to see his Home workspace even though it doesn't exists
-		// => test the get verb on the clusterworkspaces/workspace subresource named ~ in the root workspace.
+		lcluster.Name = homeLogicalClusterName
```

Comment:
Why don't we use the `searchForWorkspaceAndRBACInLocalInformers` function directly here in this case as well, and also send a retry when it should be done (typically when workspace and rbac resources are already created, but Home workspace is still initializing) ?

Reply:
This probably also works, but I assume is less optimized.

Reply:
The current code path is proven to work. This is to unblock 0.6. We can improve later in a follow-up.

Also note that searchForWorkspaceAndRBACInLocalInformers is not waiting for readiness. That was the root cause for the prior PR here. So it is not that simple.